streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -213,7 +213,8 @@ private void runSimpleCopyTest(final int numberOfRestarts,
         final KStream<Long, Long> input = builder.stream(inputTopic);
         KStream<Long, Long> output = input;
         if (throughTopic != null) {
-            output = input.through(throughTopic);
+            input.to(throughTopic);
+            output = builder.stream(throughTopic);
```

Comment:
Using `to()` and `steam()` is "simpler" as we cleanup topics in-between (and thus avoid internal topics).

We could of course also use `repartition()`.